src/iterateJsdoc.js

```diff
@@ -157,6 +159,8 @@ const getBasicUtils = (context, {tagNamePreference, mode}) => {
     return ret;
   };
 
+  utils.commentParser = parseComment;
```

Comment:
Not sure if we really want to expose that, but I did it to use the same method to parse a new comment to get the correct alignments for comparison.

Reply:
Works for me!